packages/curriculum-parser-json/index.js

```diff
@@ -0,0 +1,130 @@
+const {
+  getParser: getMarkdownParser,
+} = require('@enkidevs/curriculum-parser-markdown')
+const { sectionNames, contentTypes } = require('@enkidevs/curriculum-helpers')
+const unistBuilder = require('unist-builder')
+const unistAssert = require('unist-util-assert')
+const { safeDump } = require('js-yaml')
+
+function getParser(type) {
+  function parse(json) {
+    return Promise.resolve(parseSync(json))
+  }
+
+  function parseSync(json) {
+    switch (type) {
+      case contentTypes.EXERCISE:
+      case contentTypes.INSIGHT:
+        return insightParser(json)
+      default:
+        throw new Error(`Invalid parser type [${type}] requested`)
+    }
+  }
+
+  return {
+    parse,
+    parseSync,
+  }
+}
+
+function insightParser(json) {
+  return unistBuilder('root', buildChildren(json))
+}
+
+function buildChildren(json) {
+  // these need to be in a specific order
+  // also ignoring 'Game Content' for now
+  const children = [
+    'Content',
+    'Practice',
+    'Revision',
+    'Quiz',
+    'Exercise',
+    'Footnotes',
+  ]
```

Comment:
Actually, I don't think we have a requirement that these must be in a specific order.